tests/ddl_csc2.test/t01_create_table.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS t4;
+DROP TABLE IF EXISTS t5;
+DROP TABLE IF EXISTS t6;
+
+CREATE TABLE t4(x1 SMALLINT, x2 BIGINT, x3 INT, x4 LARGEINT);$$
```

Comment:
So, both bigint and largeint map to largeint. This is intentional. Right?

Reply:
yes